src/ts/DeviceMetadata.ts

```diff
@@ -0,0 +1,4 @@
+
+export interface DeviceMetadata {
```

Comment:
I would move this to the `Html5Metadata.ts` file as those belong together. Just add this `export` on top (after the imports)